src/coreclr/src/vm/ClrEtwAll.man

```diff
@@ -7125,6 +7154,11 @@
                 <string id="RuntimePublisher.TieredCompilationSettingsFlags.NoneMapMessage" value="None" />
                 <string id="RuntimePublisher.TieredCompilationSettingsFlags.QuickJitMapMessage" value="QuickJit" />
                 <string id="RuntimePublisher.TieredCompilationSettingsFlags.QuickJitForLoopsMapMessage" value="QuickJitForLoops" />
+                <string id="RuntimePublisher.KnownPathSource.TrustedPlatformAssembliesMessage" value="TrustedPlatformAssemblies" />
```

Comment:
There's a discussion that we should slowly start renaming TPA to something sensible: https://github.com/dotnet/runtime/pull/690#discussion_r355596277

Reply:
Good call. This one does at least have the advantage of not actually being part of the raw data / format that is traced, since it is just a string for the value map.